rca/settings/base.py

```diff
@@ -712,23 +712,11 @@
 API_CONTENT_BASE_URL = env.get("API_CONTENT_BASE_URL", "https://rca.ac.uk")
 DATA_UPLOAD_MAX_NUMBER_FIELDS = 2000
 
-# Access planit details, for fetching xml data on the shourt course pages.
-try:
-    ACCESS_PLANIT_XML_FEED_TIMEOUT = int(env.get("ACCESS_PLANIT_XML_FEED_TIMEOUT"))
-except TypeError:
-    ACCESS_PLANIT_XML_FEED_TIMEOUT = 60 * 60 * 15
-
-ACCESS_PLANIT_XML_BASE_URL = env.get("ACCESS_PLANIT_XML_BASE_URL", None)
-ACCESS_PLANIT_XML_COURSE_URL = env.get("ACCESS_PLANIT_XML_COURSE_URL", None)
-ACCESS_PLANIT_SCHOOL_ID = env.get("ACCESS_PLANIT_SCHOOL_ID")
-ACCESS_PLANIT_REGISTER_INTEREST_BASE = env.get(
-    "ACCESS_PLANIT_REGISTER_INTEREST_BASE", None
-)
```

Comment:
**Reminder**
Just a reminder to remove these on the Heroku apps

Reply:
Its on the ticket :+1: